apache2/re.c

```diff
@@ -408,30 +402,22 @@ char *update_rule_target_ex(modsec_rec *msr, msre_ruleset *ruleset, msre_rule *r
                 }
 
                 opt = strchr(param,':');
-
-                if(opt != NULL) {
+                if (opt != NULL) {
                     name = apr_strtok(param,":",&value);
                 } else {
                     name = param;
                 }
 
-                if(apr_table_get(ruleset->engine->variables, name) == NULL)   {
-                    if(target_list != NULL)
-                        free(target_list);
-                    if(replace != NULL)
-                        free(replace);
-                    if(msr) {
-                        msr_log(msr, 9, "Error to update target - [%s] is not valid target", name);
```

Comment:
Same as above (in line 299. in old version)

Reply:
Same answer as above